tests/models/whisper/test_modeling_whisper.py

```diff
@@ -89,6 +89,14 @@ def prepare_whisper_inputs_dict(
     }
 
 
+def load_datasamples(num_samples):
```

Comment:
Ideally, this sort of cosmetic change should be avoided unless there is a strong reason for it. It pollutes the diff and makes tracking changes harder :)

Reply:
got it, will remove